### PR TITLE
AARA-329: fix for lengthy names in rfa side bar

### PR DESCRIPTION
**Pull request by lauherkcambria**
#### Which User story does this relate to (link)?
-https://osi-cwds.atlassian.net/browse/AARA-329

#### Brief feature description
- names were being pulled right if they were too long, now they will be properly aligned.

#### Does this PR block/effect other stories?
-no

#### Do any commands needs to be run in order to run the project with this PR?
-no

#### Code coverage percent diff as a result of this PR?
-n/a

### CODE REVIEW

**Comment by lauherkcambria** [comment]
jenkins, retest this please.

**Comment by anuroop15** [comment]
jenkins retest this please

**Comment by lauherkcambria** [comment]
jenkins, retest this please.

File: spec/acceptance/rfa_forms/rfa_01b_application_spec.rb [hunk 1]
```diff
@@ -8,6 +8,7 @@
 
   scenario 'validate rfa01b creation and fields', set_auth_header: true do
     visit root_path
+    page.driver.browser.manage.window.resize_to 1200, 800
```
Comment (anuroop15) [inline spec/acceptance/rfa_forms/rfa_01b_application_spec.rb]: As the browser is created and ready to use with command 'visit root_path', will resizing the window work? I was not sure about this.

**Pull request merged**